src/gui/socketapi/socketapi.cpp

```diff
@@ -952,8 +952,9 @@ bool OCC::SocketApi::FileData::isSyncFolder() const
 
 SyncFileStatus SocketApi::FileData::syncFileStatus() const
 {
-    if (!folder)
+    if (!folder || folder->canSync()) {
```

Comment:
Shouldn't that be `!folder->canSync()` ? Otherwise I don't understand what's happening.

Reply:
🙈  thx